bundles/binding/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/KM200Comm.java

```diff
@@ -379,12 +377,12 @@ public void initObjects(String service) {
 
                 case "floatValue": /* Check whether the type is a single value containing a float value */
                     logger.debug("initDevice: type float value: {}", decodedData);
-                    valObject = Float.parseFloat(nodeRoot.get("value").toString());
+                    valObject = BigDecimal.valueOf(nodeRoot.getDouble("value"));
```

Comment:
Not a reason to require a change to this PR, but if you could get the value from JSON without passing it through `double`, as in getting it as the string as it appears in the JSON payload, then the `BigDecimal` would be able to represent it exactly.  By becoming a `double`, it could lose the accuracy as it appears in JSON.  However, this JSON library may not let you do this.